servers/rendering/renderer_rd/effects_rd.h

```diff
@@ -782,7 +782,7 @@ class EffectsRD {
 	void make_mipmap(RID p_source_rd_texture, RID p_dest_texture, const Size2i &p_size);
 	void copy_cubemap_to_dp(RID p_source_rd_texture, RID p_dest_texture, const Rect2 &p_rect, float p_z_near, float p_z_far, bool p_dp_flip);
 	void luminance_reduction(RID p_source_texture, const Size2i p_source_size, const Vector<RID> p_reduce, RID p_prev_luminance, float p_min_luminance, float p_max_luminance, float p_adjust, bool p_set = false);
-	void luminance_reduction_raster(RID p_source_texture, const Size2i p_source_size, const Vector<RID> p_reduce, Vector<RID> p_fb, RID p_prev_luminance, float p_min_luminance, float p_max_luminance, float p_adjust, bool p_set = false);
+	void luminance_reduction_raster(RID p_source_texture, const Size2i p_source_size, const Vector<RID> p_reduce, Vector<RID> p_fb, RID p_prev_luminance, float p_min_luminance, float p_max_luminance, float p_adjust, bool p_set = false, float p_color_scale = 2.0);
```

Comment:
does it make sense to scale the luminance reduction? I would just multiply by scale in the tonemapper, so no precision is lost.

Reply:
We render into a single channel buffer here so there is alot more precision then the source color. I guess we could do the multiplication much later

Reply:
I've moved this into the tonemapper, there isn't any difference performance wise but doing this in one place instead of two makes sense.